csv.h

```diff
@@ -753,30 +787,26 @@ namespace io{
                 }
         };
 
-        template<char sep>
```

Comment:
Where did no_quote_escape go? Removing this is a no-go. 90% of CSV files are just numbers without quotes. Parsing these must be fast and therefore the default policy setting should not involve any quote handling. Without this policy, this is no longer possible.

Reply:
It was moved up since some compilers are giving error for the declaration location.
this change can be omitted